dwh/indicator_lib.py

```diff
@@ -150,7 +84,7 @@ def calc_TX_PVLS(patient_agg_obs: DataFrame, VL_code: str, failure_threshold: in
   if end_date_str:
     end_date = datetime.strptime(end_date_str, '%Y-%m-%d')
   agg_buckets_udf = F.UserDefinedFunction(
-      lambda a, g: agg_buckets(a, g, end_date),
+      lambda a, g: _agg_buckets(a, g, end_date),
       T.ArrayType(T.StringType()))
   VL_df = patient_agg_obs.withColumn(
       'sup_VL', patient_agg_obs[VL_code + '_max_value'] < failure_threshold).withColumn(
```

Comment:
`.withColumn` is a spark API, I'm not sure if pandas support the same. Maybe [assign](https://pandas.pydata.org/pandas-docs/version/0.23/generated/pandas.DataFrame.assign.html) might be appropriate here.